merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
So if I do this:

```
jobs:
- (( delete "something" ))
```

and there is no job `something`, the behavior is going to change to _not_ fail?